File: snippets/fsharp/System/Math/BigMul/bigmul.fs
module bigmul

//<snippet1>
// This example demonstrates Math.BigMul()
open System

let int1 = Int32.MaxValue
let int2 = Int32.MaxValue

let longResult = Math.BigMul(int1, int2)
printfn "Calculate the product of two Int32 values:"
printfn $"{int1} * {int2} = {longResult}"

// This example produces the following results:
// Calculate the product of two Int32 values:
// 2147483647 * 2147483647 = 4611686014132420609
//</snippet1>

File: snippets/csharp/System/Math/BigMul/bigmul.cs
//<snippet1>
// This example demonstrates Math.BigMul()
using System;

class Sample
{
    public static void Main()
    {
        int int1 = int.MaxValue;
        int int2 = int.MaxValue;
        long longResult;
        //
        longResult = Math.BigMul(int1, int2);
        Console.WriteLine("Calculate the product of two Int32 values:");
        Console.WriteLine($"{int1} * {int2} = {longResult}");
    }
}
/*
This example produces the following results:
Calculate the product of two Int32 values:
2147483647 * 2147483647 = 4611686014132420609
*/
//</snippet1>
